packages/office-ui-fabric-react/src/components/Button/Button.types.ts

```diff
@@ -185,6 +186,22 @@ export interface IButtonProps extends React.AllHTMLAttributes<HTMLAnchorElement
  * Any custom data the developer wishes to associate with the menu item.
  */
   data?: any;
+
+  /**
+  * Method to provide the classnames to style a button.
+  * The default value for this prop is the getClassnames func
+  * defined in BaseButton.classnames.
+  * @default getBaseButtonClassNames
+  */
+  getClassNames?: (styles: IButtonStyles,
```

Comment:
I wonder if we should not pass in the styles for getClassNames (like we do for contextual menu). If you are implementing getClassNames, you'll likely want to provide the styles yourself

Reply:
Yes I agree... let me remove them